engine/src/main/kotlin/io/provenance/scope/ContractEngine.kt

```diff
@@ -26,6 +26,7 @@ import io.provenance.scope.objectstore.util.toHex
 import io.provenance.scope.objectstore.util.toPublicKeyProtoOS
 import io.provenance.scope.util.ContractDefinitionException
 import io.provenance.scope.util.ProtoUtil.proposedRecordOf
+import io.provenance.scope.util.forThread
```

Comment:
unused?

Reply:
no, it is used, I just moved this to an extension function from within util for use in the CachedOsClient